src/js/components/Button/StyledButtonKind.js

```diff
@@ -68,36 +79,69 @@ const basicStyle = props => css`
   }
 `;
 
+// could replace this with a call in lodash
+const getPath = (theme, path) => {
+  let obj;
+  if (path) {
+    obj = theme;
+    const parts = path.split('.');
+    while (obj && parts.length) obj = obj[parts.shift()];
+  }
+  return obj;
+};
+
+const dimensionToNumber = d => parseInt(d.replace('px', ''), 10);
```

Comment:
Can we use the existing `parseMetricToNum`?

Reply:
Looks like I probably can. I didn't think it would strip off the units.

Reply:
replaced with parseMetricToNum